apps/mobile/src/components/Markdown.tsx

```diff
@@ -10,7 +10,7 @@ const markdownStyles = StyleSheet.create({
   body: {
     fontSize: 12,
     color: '#141414',
-    fontFamily: 'ABCDiatypeRegular',
+    // fontFamily: 'ABCDiatypeRegular',
```

Comment:
@tbezman Do you mind if I removed this? Used it for highlighted text in search result